api/config/service/aws/v1/aws.proto

```diff
@@ -10,6 +10,11 @@ message Config {
   repeated string regions = 1 [ (validate.rules).repeated = {min_items : 1} ];
   ClientConfig client_config = 2;
   DynamodbConfig dynamodb_config = 3;
+
+  // The current account alias, if this is not set the default will be "default"
+  string account_alias = 4;
```

Comment:
Is this the primary account?

Reply:
Yeah this is the primary account, maybe we rename to include that? `primary_account_alias`?